### DISCUSSION TITLE
Webhooks

### DISCUSSION THREAD

**Discussion by cuboooooo** [maximmax42/Discord-CustomRP#49] [Q&A]
I'm not sure how hard it is, but could you possibly turn on Webhooks? https://magefan.com/blog/configure-webhooks-in-github I think that might work. Just a thought ¯\\_(ツ)_/¯

**Comment by maximmax42** [comment]
Uhhhh. You mean for the repository?

**Reply by BirbEXE** [reply]
think that's what he means. not sure why tho
